docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1190,84 +1130,45 @@ function setAudioOutputDevice() {
     });
 }
 
+leet = audioEnabled = true;
```

Comment:
is this a typo for `let audioEnabled`?

Reply:
yeah, typo and anyway `audioEnabled` was not used, so removed it